pkg/virtualization/core/service/dvddrive.go

```diff
@@ -84,27 +84,6 @@ func (vmms *VirtualSystemManagementService) AddISODisk(vm *virtualsystem.Virtual
 
 }
 
-func (vmms *VirtualSystemManagementService) RemoveISODisk(ld *disk.LogicalDisk) (err error) {
-	dvddrive, err := ld.GetDrive()
```

Comment:
Any reason to remove this method. These are used in unit tests

Reply:
This was done by mistake. Let me add it back.